qownnotes/snapcraft.yaml

```diff
@@ -10,7 +10,7 @@ description: |
 
 apps:
   qownnotes:
-    command: desktop-launch usr/bin/QOwnNotes --snap
+    command: desktop-launch ${SNAP}/usr/bin/QOwnNotes --snap
```

Comment:
@pbek the internal path are prepended: export PATH="$SNAP/bin:$SNAP/usr/bin:$PATH" in the generated wrapper.